common/history/buildings/07_south_america.txt

```diff
@@ -1,4 +1,4 @@
-﻿BUILDINGS={
```

Comment:
TWT has -182 bureaucracy. Is there a reason for this?

Reply:
It's been like that for a while, and is semi-intended. Once I get working on TWT again I'll formalize it in some negative bureaucracy modifiers they'll have in the early game representing use of the quipu, which were knotted strings used in place of a writing system.